tools/c7n_azure/c7n_azure/session.py

```diff
@@ -18,13 +18,15 @@
 from azure.cli.core.cloud import AZURE_PUBLIC_CLOUD
 from azure.cli.core._profile import Profile
 from azure.common.credentials import ServicePrincipalCredentials, BasicTokenAuthentication
+import adal
 
 
 class Session(object):
 
     def __init__(self):
         self.log = logging.getLogger('custodian.azure.session')
         self._provider_cache = {}
+        adal.set_logging_options({'level': 'WARNING'})
```

Comment:
is this something we want to set at session creation (its a global variable modification) or just once at the module level?

Reply:
So I considered just putting it in `__init__.py` for the Azure plugin, and I have confirmed that works fine.  Does that seem reasonable?

Reply:
sounds good to me.